### ISSUE TITLE
The first label is missing while plotting

### ISSUE THREAD

**Issue by baigal628** [pinellolab/STREAM#140]
Hi,

Thanks for developing STREAM!
I have a question about the stream plotting function. In my case, there seems to be one label legend missing every time.

When colored by label, '0_Naive' corresponding to red colors is missing. When colored by annotation, the legend for blue colors is missing. I am not sure what causes this problem. Any suggestion to trace the error?

Thanks!

```{python}
adata = scanpy.read_h5ad("./data/cd3_pos_cd4_neg_sub.h5ad")
st.set_workdir(adata,'./stream_result')
st.add_cell_labels(adata, file_name='./data/cd3_pos_cd4_neg_sub_2000_label.tsv')
st.add_cell_colors(adata)
adata.obsm['top_pcs'] = adata.obsm['X_pca']
adata.obsm['X_dr'] = adata.obsm['X_pca']
adata.obsm['X_vis_umap'] = adata.obsm['X_umap'][:,:2]
```

```{python}
st.plot_dimension_reduction(adata,color=['label'],n_components=2,show_graph=True,show_text=True)
st.plot_dimension_reduction(adata,color=['annotation'],n_components=2,show_graph=True,show_text=True)
```
<img width="510" alt="image" src="https://user-images.githubusercontent.com/56844380/179623464-4a199477-1550-4e06-b3ce-9e99de56e706.png">
<img width="485" alt="image" src="https://user-images.githubusercontent.com/56844380/179623478-ff409cb8-60d1-4021-b84b-3dabf6b03f97.png">

**Comment by huidongchen** [comment]
Hi thanks for the feedback! The issue appears similar to #130 . Are you running the latest version v1.1? If not, can you give it a try to see if that helps?

**Comment by baigal628** [comment]
Thanks! Fixed after upgrading to v1.1.
